app/models/solr_document.rb

```diff
@@ -502,4 +502,28 @@ def md5
     # this nonsensical ':symbol' option indicates that I am selecting the _ssim suffix from down in solrizer - default was _tesim, which was wrong for this field
     self[Solrizer.solr_name('md5', :symbol)]
   end
+
+  def all_nested_member_ids
+    all_member_ids = self.member_ids.clone
+    self.member_ids.each do |member_id|
+      all_member_ids << SolrDocument.get_members(member_id)
+    end
+    all_member_ids.flatten
+  end
+
+  private
+
+  # Recursively get all members off of members
+  def self.get_members(id)
+    ids = []
+    object = SolrDocument.find(id)
+    object.member_ids.each do |member_id|
+      ids << member_id
+      SolrDocument.find(member_id).member_ids.each do |mem_id|
+        get_members(mem_id)
+      end
+    end
+    ids
+  end
+
```

Comment:
@jasoncorum on another look, I don't think this recursive method would work as expected. The method returns the `ids` array, but the recursive call on line 523 does not assign the return value to anything. Also, the nested loop means you go from parent-to-grandchild before recursing, but i think we want to recurse at every level.

What about something like this?
```
def self.get_members(id, fam = [])
  fam << id
  id.member_ids.each { |member_id| get_members(member_id, fam) }
  fam
end
```